Directory.Packages.props

```diff
@@ -2,10 +2,10 @@
 
   <ItemGroup>
     <PackageVersion Include="Expecto" Version="[9.0, 10.0)" />
-    <PackageVersion Include="FSharp.Core" Version="[4.6.0,)" />
+    <PackageVersion Include="FSharp.Core" Version="[6.0,)" />
     <PackageVersion Include="Microsoft.NET.Test.Sdk" Version="17.1.0" />
     <PackageVersion Include="Microsoft.TestPlatform.ObjectModel" Version="17.1.0" />
-    <PackageVersion Include="System.Collections.Immutable" Version="[1.7.1,)" />
+    <PackageVersion Include="System.Collections.Immutable" Version="[6.0,)" />
```

Comment:
Why is this upgraded? And do we need the newest version?

Reply:
Lates Visual Studio loads it anyway

Reply:
![image](https://user-images.githubusercontent.com/2365592/161640432-e4348395-be37-41d9-b62b-2b634edcadce.png)
![image](https://user-images.githubusercontent.com/2365592/161640528-a3506c53-a5e1-4c5b-ae45-976af6cc3956.png)

Looks like tests run in a separate process, don't they?

Reply:
Tests should run in a separate process - yes. Also I would prefer this to still work in VS 2019 if possible - are you able to test that?

Reply:
I don't have it on my machine. Do you have one?

Reply:
I do not. But then again, I care more about the fact that it's working on CI, than in VS.